flutter_local_notifications_windows/lib/src/details/notification_audio.dart

```diff
@@ -101,41 +98,22 @@ class WindowsNotificationAudio {
   })  : isSilent = false,
         source = sound.name;
 
-  /// Audio from a file. See [allowedSchemes] and [allowedExtensions].
-  WindowsNotificationAudio.fromFile({
-    required Uri file,
+  /// Uses an audio file from a Flutter asset.
+  ///
+  /// Note that this will only work in release builds that have been packaged as
+  /// an MSIX installer. Currently, there is no way to test this in debug mode.
+  ///
+  /// Windows supports the following formats: `.aac`, `.flac`, `.m4a`, `.mp3`,
+  /// `.wav`, and `.wma`.
+  WindowsNotificationAudio.asset(
+    String assetName, {
     this.shouldLoop = false,
+    WindowsNotificationSound debugModeFallback =
```

Comment:
Since `debugModeFallback` isn't a property, can you update the constructor's API docs to explain what `debugModeFallback` is used for? I believe that's the only place to do so as I tried using `///` above the line and it wasn't included in the generated docs. If it helps copilot came up with `The [debugModeFallback] sound will be used in debug mode.` that can be added and this was the result

<img width="893" alt="image" src="https://github.com/user-attachments/assets/ae4d5388-366d-44e1-ab17-5f74f641e5ad" />

Reply:
(Done)